assets/style.css

```
@@ -45,6 +41,24 @@
     border-top: 0 !important;
 }
 
+body.rex-theme-dark .rex-ytraduko-overview thead tr > th {
+    background: #202b35;
```

```suggestion
    background-color: #202b35;
```

Beim Theming möglichst nur die genaue Eigenschaft überschreiben, dann bleiben bei Bedarf begleitende Eigenschaften erhalten (hier z. B. `-image`, `-postion`, `-repeat`).